trin-validation/src/accumulator.rs

```diff
@@ -37,11 +43,16 @@ pub struct MasterAccumulator {
 impl MasterAccumulator {
     /// Load default trusted master acc
     pub fn try_from_file(master_acc_path: PathBuf) -> anyhow::Result<MasterAccumulator> {
-        let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-        path.push(master_acc_path);
-        let raw = std::fs::read(&path)
-            .map_err(|_| anyhow!("Unable to find master accumulator at path: {:?}", path))?;
-        MasterAccumulator::from_ssz_bytes(&raw)
+        let raw = match TrinValidationAssets::get(&(*master_acc_path).display().to_string()[..]) {
+            Some(raw) => Ok(raw),
+            None => Err({
+                anyhow!(
+                    "Unable to find master accumulator at path: {:?}",
+                    master_acc_path
+                )
+            }),
+        }?;
```

Comment:
```suggestion
        let raw = TrinValidationAssets::get(&(*master_acc_path).display().to_string()[..])
            .map_err(|_| anyhow!("Unable to find master accumulator at path: {master_acc_path:?}"))?;
```

Reply:
.map_err cant be used on an option apparently it can only be used on a result.

Reply:
The match cases does what map_err() would do though

Reply:
Ahh ok, let's use  [ok_or](https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or), this will transform the option to Result and we can return the error with `?`.

Reply:
that is a very cool function I didn't know about ty for letting me know :). Resolved